model_compression_toolkit/exporter/model_wrapper/keras/builder/node_to_quantizer.py

```diff
@@ -24,54 +26,59 @@
-            return {qi_keras_consts.NUM_BITS: node_w_qc.weights_n_bits,
-                    qi_keras_consts.PER_CHANNEL: node_w_qc.weights_per_channel_threshold,
-                    qi_keras_consts.MIN_RANGE: list(node_w_qc.weights_quantization_params[RANGE_MIN].flatten()),
-                    qi_keras_consts.MAX_RANGE: list(node_w_qc.weights_quantization_params[RANGE_MAX].flatten()),
-                    qi_keras_consts.CHANNEL_AXIS: node_w_qc.weights_channels_axis,
-                    qi_keras_consts.INPUT_RANK: len(node_w_qc.weights_quantization_params[RANGE_MIN].shape)}
+            return {qi_keras_consts.NUM_BITS: node_qc.weights_n_bits,
+                    qi_keras_consts.PER_CHANNEL: node_qc.weights_per_channel_threshold,
+                    qi_keras_consts.MIN_RANGE: list(node_qc.weights_quantization_params[RANGE_MIN].flatten()),
+                    qi_keras_consts.MAX_RANGE: list(node_qc.weights_quantization_params[RANGE_MAX].flatten()),
+                    qi_keras_consts.CHANNEL_AXIS: node_qc.weights_channels_axis,
+                    qi_keras_consts.INPUT_RANK: len(node_qc.weights_quantization_params[RANGE_MIN].shape)}
 
         elif quantization_method in [QuantizationMethod.LUT_SYM_QUANTIZER, QuantizationMethod.LUT_POT_QUANTIZER]:
-            return {qi_keras_consts.NUM_BITS: node_w_qc.weights_n_bits,
-                    qi_keras_consts.PER_CHANNEL: node_w_qc.weights_per_channel_threshold,
-                    qi_keras_consts.CLUSTER_CENTERS: node_w_qc.weights_quantization_params[CLUSTER_CENTERS],
-                    qi_keras_consts.THRESHOLD: list(node_w_qc.weights_quantization_params[SCALE_PER_CHANNEL].flatten()),
-                    qi_keras_consts.CHANNEL_AXIS: node_w_qc.weights_channels_axis,
+            return {qi_keras_consts.NUM_BITS: node_qc.weights_n_bits,
+                    qi_keras_consts.PER_CHANNEL: node_qc.weights_per_channel_threshold,
+                    qi_keras_consts.CLUSTER_CENTERS: node_qc.weights_quantization_params[CLUSTER_CENTERS],
+                    qi_keras_consts.THRESHOLD: list(node_qc.weights_quantization_params[SCALE_PER_CHANNEL].flatten()),
+                    qi_keras_consts.CHANNEL_AXIS: node_qc.weights_channels_axis,
                     # TODO: how to pass multiplier nbits and eps for a specific node?
-                    qi_keras_consts.INPUT_RANK: len(node_w_qc.weights_quantization_params[SCALE_PER_CHANNEL].shape)}
+                    qi_keras_consts.INPUT_RANK: len(node_qc.weights_quantization_params[SCALE_PER_CHANNEL].shape)}
 
         else:
             Logger.critical(f'Not supported quantization method for inferable quantizers.')  # pragma: no cover
 
     elif quantization_target == QuantizationTarget.Activation:
-        # Get the activation quantization configuration for the node
-        node_qc = node.final_activation_quantization_cfg
+        if not isinstance(node_qc, NodeActivationQuantizationConfig):
```

Comment:
1. No need for the same check in the weight quantizer section?
2. add no-cover pragma

Reply:
1. There is (line 44)
2. V